udata/core/organization/views.py

```diff
@@ -291,6 +296,26 @@ def datasets_csv(org):
     return csv.stream(adapter, '{0}-datasets'.format(org.slug))
 
 
+@blueprint.route('/<org:org>/issues.csv')
+def issues_csv(org):
+    datasets = Dataset.objects.filter(organization=str(org.id))
+    issues = [DatasetIssue.objects.filter(subject=dataset)
+              for dataset in datasets]
+    # Turns a list of lists into a flat list.
+    adapter = IssuesOrDiscussionCsvAdapter(itertools.chain(*issues))
+    return csv.stream(adapter, '{0}-issues'.format(org.slug))
+
+
+@blueprint.route('/<org:org>/discussions.csv')
+def discussions_csv(org):
+    datasets = Dataset.objects.filter(organization=str(org.id))
+    discussions = [DatasetDiscussion.objects.filter(subject=dataset)
```

Comment:
I think ``DatasetDiscussion.objects(subject__in=datasets)`` should be more efficient